### PR TITLE
Tutorial1

### PR DESCRIPTION
**Pull request by usmanazFIU**

### CODE REVIEW

File: tutorials/napps/development_environment_setup.rst [hunk 1]
```diff
@@ -48,48 +48,10 @@ Install the dependencies necessary to build Python:
 
 .. code-block:: console
 
+ sudo add-apt-repository ppa:deadsnakes/ppa
  sudo apt update
- sudo apt install build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libreadline-dev libffi-dev libsqlite3-dev wget libbz2-dev
+ sudo apt install python3.9
```
Comment (viniarck) [inline tutorials/napps/development_environment_setup.rst:53]: @usmanazFIU did you have the chance to confirm if it also installs pip and the dev dependencies? I wonder if `python3.9-dev python3.9-venv` would also be needed.

File: tutorials/napps/development_environment_setup.rst [hunk 2]
```diff
@@ -310,6 +272,23 @@ To test if the mininet is working for you, run the command:
...
 
+How to use with MongoDB
+=======================
+
+Kytos with MongoDB requires docker and docker-compose to be installed on your linux environment. The following tutorials have been tested on Ubuntu 20.0.4 for docker and docker-compose
+
+Installing docker
+-----------------
+
+https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-20-04 (Follow till Step 3 for installing required docker dependencies)
```
Comment (viniarck) [inline tutorials/napps/development_environment_setup.rst:283]: The instructions and links look great, @usmanazFIU, could you reformat them using RST links instead? Either inline or reference links would be fine I think.

Comment (usmanazFIU) [reply tutorials/napps/development_environment_setup.rst]: ive updated

File: tutorials/napps/development_environment_setup.rst (hunk 1 above)
Comment (usmanazFIU) [reply tutorials/napps/development_environment_setup.rst:53]: pip is installed with python installation. for the safe side we can add sudo apt install python3.9-dev and sudo apt install 
python3.9-venv

Comment (usmanazFIU) [reply tutorials/napps/development_environment_setup.rst:53]: from the internet 
image

**Comment by viniarck** [comment]
This phrase:

```
Now that Kytos is running, switch back to the previous window and install the NApps using the github command line utility.
```

Let's make a change here and install NApps without Kytos running to simplify things:

```
Install the NApps cloning from GitHub and running python3 setup.py develop
```

**Comment by viniarck** [comment]
This part

```
now disable all napps

$ kytos napps disable all
```

I don't think we need this, maybe this could just explain that all NApps (or an individual NApp) can be disabled with this command but no need run this?

**Comment by viniarck** [comment]
mininet test command it's advisable to include `--controller`, let's use this one here, up to this part kytosd needs to be up and running:

```
mn --test pingall --controller=remote,ip=127.0.0.1,port=6653
```

**Comment by viniarck** [comment]
After all this changes, please make sure to go through the entire tutorial one more time, if it's all set, then let's get it merged since new contributors will follow this tutorial to set their environment thanks @usmanazFIU

**Review by viniarck — CHANGES_REQUESTED** [review]
Requested some changes mentioned on my previous replies.

File: tutorials/napps/development_environment_setup.rst [hunk 3]
```diff
@@ -170,6 +170,26 @@ later in the following tutorials.
 
 Install the NApps cloning from GitHub and running python3 setup.py develop
 
+Currently, flow_manager and topology requires MongoDB to be setup. so before installing the Napps follow the instructions below
```
Comment (viniarck) [inline tutorials/napps/development_environment_setup.rst:173]:
```suggestion
Currently, flow_manager and topology require MongoDB to be setup. Before installing the Napps, follow the instructions below
```

**Review by viniarck — APPROVED** [review]
Nicely done @usmanazFIU, great to have this updated tutorial, it'll facilitate for new developers.

**Pull request merged**